app/services/google/keyword_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Google
+  class KeywordService
```

Comment:
```suggestion
  class SearchService
```
In general, [services](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#services) start with a verb to describe the action that will be triggered when calling the `call` method.

As this service can be used to search anything on Google (not just a keyword but also a whole phrase), I think we do not need to add the word "keyword" in its name.

That would give us: `Google::SearchService`. :)